indico/modules/events/abstracts/controllers/management.py

```diff
@@ -159,3 +159,66 @@ def _process(self):
             return jsonify_data()
         return jsonify_form(form, skip_labels=True, form_header_kwargs={'id': 'reviewing-role-form'},
                             disabled_until_change=True)
+
+
+class RHManageAbstractReviewingQuestions(RHManageAbstractsBase):
+    def _process(self):
+        actions = {'create': 'abstracts.create_reviewing_question', 'edit': 'abstracts.edit_reviewing_question',
+                   'delete': 'abstracts.delete_reviewing_question', 'sort': 'abstracts.sort_reviewing_questions'}
+        return jsonify_template('events/reviewing_questions_management.html', event=self.event,
+                                reviewing_questions=self.event.abstract_review_questions,
+                                field_types=get_reviewing_field_types('abstracts'), actions=actions, args={})
+
+
+class RHCreateAbstractReviewingQuestion(RHManageAbstractsBase):
+    def _process_args(self):
+        RHManageAbstractsBase._process_args(self)
+        try:
+            self.field_cls = get_reviewing_field_types('abstracts')[request.args['field_type']]
+        except KeyError:
+            raise NotFound
+
+    def _process(self):
+        form = self.field_cls.create_config_form()
+        if form.validate_on_submit():
+            new_question = create_reviewing_question(self.event, AbstractReviewQuestion, self.field_cls, form)
+            self.event.abstract_review_questions.append(new_question)
+            logger.info("New abstract reviewing question %s has been created by %s", new_question, session.user)
+            return jsonify_data(flash=False)
+        return jsonify_form(form, fields=getattr(form, '_order', None))
+
+
+class RHReviewingQuestionBase(RHManageAbstractsBase):
+    locator_args = {
+        lambda self: self.question
+    }
+
+    def _process_args(self):
+        RHManageAbstractsBase._process_args(self)
+        self.question = AbstractReviewQuestion.get_one(request.view_args['question_id'])
+
+
+class RHEditAbstractReviewingQuestion(RHReviewingQuestionBase):
+    def _process(self):
+        defaults = FormDefaults(obj=self.question, **self.question.field_data)
+        form = self.question.field.create_config_form(obj=defaults)
+        if form.validate_on_submit():
+            edit_reviewing_question(self.question, form)
+            logger.info("Abstract reviewing question %s has been updated by %s", self.question, session.user)
```

Comment:
server-side log messages can be more concise; i'd remove the "has been". Also, `%r` since we use the reprs here. Makes no difference since we don't use `__str__` on our models, but st ill cleaner.

You could also move it into the operation and just log it as `Reviewing question %r ...` so it'll be fine for all types of reviewing. The repr of the question makes it clear enough what it is.